contrib/onscripter-en/template.py

```diff
@@ -0,0 +1,34 @@
+pkgname = "onscripter-en"
+pkgver = "2024.07.21"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_gen = []
+make_cmd = "gmake"
+make_dir = "."
+hostmakedepends = [
+    "gmake",
+    "gsed",
+    "pkgconf",
+]
+makedepends = [
+    "bzip2-devel",
+    "freetype-devel",
+    "libogg-devel",
+    "libvorbis-devel",
+    "libx11-devel",
+    "sdl1.2_image-devel",
+    # note: refuses to use system SDL_mixer if it's not linked with smpeg
+    # "sdl1.2_mixer-devel",
```

Comment:
link it with smpeg then (i don't see a reason to not do it; i just forgot to ask back when it was added)

Reply:
seems to only need patching configure.in to pkg-config smpeg and it would link ok

Reply:
well, and also needing smpeg-not-2